src/main/resources/alfresco/public-rest-context.xml

```diff
@@ -140,6 +140,7 @@
                 <entry key="org.alfresco.rest.framework.core.exceptions.ApiException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_INTERNAL_SERVER_ERROR}" />
                 <entry key="java.lang.Throwable" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_INTERNAL_SERVER_ERROR}" />
                 <entry key="java.lang.IllegalArgumentException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_BAD_REQUEST}" />
+                <entry key="org.alfresco.error.AlfrescoRuntimeException" value="#{T(org.springframework.extensions.webscripts.Status).STATUS_BAD_REQUEST}" />
```

Comment:
AlfrescoRuntimeExceptions are valid 500 errors. Unfortunately this change can't be done.

Reply:
Sure.  Renamed then accordingly to their module exception class.